zebra-chain/src/parameters/network/tests/vectors.rs

```diff
@@ -179,6 +179,8 @@ fn check_configured_network_name() {
         .expect_err("should panic when setting network name that's too long or contains non-alphanumeric characters (except '_')");
     }
 
+    drop(std::panic::take_hook());
```

Comment:
Why do we need the drop here?

Reply:
I thought I saw something in the logs about it failing to set a panic hook because there was already a panic hook set (when `test_database_format()` failed), but it may have actually been unrelated, since panic hooks can be overwritten, I think it was `eyre::set_hook()` that failed, so, we probably don't need the drop here, I'll revert it.